[2024-05-09T10:10:16.589+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-09T10:10:16.635+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mysql_table_dump.import_id manual__2024-05-09T10:10:11.040782+00:00 [queued]>
[2024-05-09T10:10:16.645+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mysql_table_dump.import_id manual__2024-05-09T10:10:11.040782+00:00 [queued]>
[2024-05-09T10:10:16.646+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-09T10:10:16.686+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): import_id> on 2024-05-09 10:10:11.040782+00:00
[2024-05-09T10:10:16.695+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'mysql_table_dump', 'import_id', 'manual__2024-05-09T10:10:11.040782+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/dump_load_dag.py', '--cfg-path', '/tmp/tmph8ac2pjk']
[2024-05-09T10:10:16.700+0000] {standard_task_runner.py:91} INFO - Job 11: Subtask import_id
[2024-05-09T10:10:16.706+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=230) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-09T10:10:16.707+0000] {standard_task_runner.py:63} INFO - Started process 231 to run task
[2024-05-09T10:10:16.783+0000] {task_command.py:426} INFO - Running <TaskInstance: mysql_table_dump.import_id manual__2024-05-09T10:10:11.040782+00:00 [running]> on host 808b91199023
[2024-05-09T10:10:17.003+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mysql_table_dump' AIRFLOW_CTX_TASK_ID='import_id' AIRFLOW_CTX_EXECUTION_DATE='2024-05-09T10:10:11.040782+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-09T10:10:11.040782+00:00'
[2024-05-09T10:10:17.005+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-09T10:10:17.025+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-09T10:10:17.026+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'mysql -u root -proot -h host.docker.internal -P 3306 client_rw </opt/***/dags/dump_file.sql ']
[2024-05-09T10:10:17.039+0000] {subprocess.py:86} INFO - Output:
[2024-05-09T10:10:17.844+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-09T10:10:17.846+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-09T10:10:17.875+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=mysql_table_dump, task_id=import_id, execution_date=20240509T101011, start_date=20240509T101016, end_date=20240509T101017
[2024-05-09T10:10:17.925+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-09T10:10:17.943+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-09T10:10:17.945+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
